src/app/components/app.tsx

```diff
@@ -205,33 +189,19 @@ export const App = (props: IAppProps) => {
 
   const lang = getDefaultLanguage();
 
-  const modelState = useModelState<IModelInputState, IModelOutputState>(useMemo(() => ({
-    initialInputState: interactiveState?.inputState || defaultInitialState.initialInputState,
-    initialOutputState: interactiveState?.outputState || defaultInitialState.initialOutputState,
-    initialModelRuns: interactiveState?.modelRuns || defaultInitialState.initialModelRuns,
-  }), [interactiveState, defaultInitialState.initialInputState, defaultInitialState.initialOutputState, defaultInitialState.initialModelRuns]));
+  const modelState = useModelState(useMemo(() => ({
+    initialInputState: interactiveState?.inputState || defaultInitialState.inputState!,
```

Comment:
See comment in `types.ts` about removing the `Partial<>` which should let you get rid of the `!` here which is causing the lint error.

Reply:
Good call, thank you!